hoa2dot/core.py

```diff
@@ -333,8 +408,13 @@ class FalseLabelExpression(LabelExpression):
     """This class represent a "never accepting" condition."""
 
     def __str__(self):
+        """Transform the FalseLabelExpression to string."""
         return "f"
 
+    def __eq__(self, other):
+        """Check equality between two FalseLabelExpressions."""
+        return isinstance(other, FalseLabelExpression) and self.__str__() == other.__str__()
```

Comment:
`self.__str__()` and `other.__str__()` can be replaced by `str(self)` and `str(other)`